app/controllers/articles_controller.rb

```diff
@@ -30,6 +34,14 @@ def update
 
   private
 
+  def add_category_to_article
+      categories = params[:article][:categories]
+      categories.each do |category_id|
+        category = Category.find_by(id: category_id)
+        @article.categories << category
```

Comment:
I would assume you need to save the instance once you added the categories, right?

Reply:
We save it on line 8

Reply:
Okay, thx.